Automattic/simplenote-ios
# Update screenshots automation with new details and iOS version – Local only

### Fix

Updates the screenshots automation to work with ~~Xcode 12.3 and the iOS 14.3 Simulators~~ Xcode 12.4 and the iOS 14.4 Simulators.

It also replaces the screenshot for the collaborate screen with one for the inter-note linking features. Internal ref p2XJRt-2wR.

~~This is off `release/4.30` because I want the new screenshots to go in with that release.~~

### Test

I'm having issues on CI, because of the [known `xcodebuild` bug with SPM and SSH repositories](https://discuss.bitrise.io/t/xcode-11-resolving-packages-fails-with-ssh-fingerprint/10388/2). To keep the work on this PR from taking too long, I decided to drop the CI update part. I think this is acceptable, albeit undesirable because we rarely updates the screenshot.

To test locally, you can either:

- Running the "SimplenoteScreenshots" scheme; it should pass
- Running `bundle exec fastalne take_screenshots`; this will take a while and the result should be:

![screencapture-file-Users-gio-Developer-a8c-snios-fastlane-screenshots-screenshots-html-2021-01-15-20_54_45](https://user-images.githubusercontent.com/1218433/104710233-f77dbb80-5773-11eb-9b59-71b3bbac9fdb.png)

In particular, notice the new screenshot with the inter note linking:

![image](https://user-images.githubusercontent.com/1218433/103731498-630bae80-5039-11eb-8a05-7a4dc2e9c4ac.png)

and the one with the new passcode screen:

![image](https://user-images.githubusercontent.com/1218433/104710366-1aa86b00-5774-11eb-884d-8aeb1706e387.png)

If you run the screenshots composition with 

```
bf create_promo_screenshots source:$(pwd)/fastlane/screenshots/
```

You should see up-to-date copy and background, as per internal discussion, ref p2XJRt-2wR.

### Review
Only one platform developer required to review these changes, but anyone can perform the review.

### Release

These changes do not require release notes.
